src/config.template.cjs

```diff
@@ -21,6 +21,22 @@ module.exports = {
 	 */
 	torznab: [],
 
+	/**
+	 * URL to your Sonarr instance, in the same format URL as a torznab URL
+	 * but for your Sonarr.
+	 *
+	 * Example: sonarr: "http://sonarr:8989/api?apikey=12345",
+	 */
+	sonarr: undefined,
```

Comment:
Should it specify that `apikey` is your Sonarr API key? Just to be explicitly clear? Same with the line below.